### ISSUE TITLE
Doesn't work on Mac OOTB (solution provided)

### ISSUE THREAD

**Issue by Marx2** [cassiomolin/log-aggregation-spring-boot-elastic-stack#1]
To make example working on Mac I had to add memory setting to elastic search:
      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
Solution based on:
https://github.com/elastic/elasticsearch-docker/issues/22

**Comment by devfcr** [comment]
Yes, as Marx2 said, adding the memory setting to the docker-compose configuration for the elasticsearch docker container solved the issue:
`  elasticsearch:
    image: docker.elastic.co/elasticsearch/elasticsearch:7.2.0
    ports:
      - "9200:9200"
    environment:
      - "discovery.type=single-node"
      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
    volumes:
      - ./elasticsearch/data:/usr/share/elasticsearch/data:rw              # Persistence data`
